src/components/UI/IssueOrRevokeSelector/IssueOrRevokeSelector.tsx

```diff
@@ -1,12 +1,20 @@
 import { FunctionComponent } from "react";
 import { NavLink } from "react-router-dom";
 
-export const IssueOrRevokeSelector: FunctionComponent = () => (
-  <div className="space-x-6 text-cloud-900 text-lg mb-4">
-    <NavLink to="/forms-selection" activeClassName="font-bold text-blue underline">
+interface IssueOrRevokeSelectorProps {
+  createLink?: string;
+  revokeLink?: string;
+}
```

Comment:
why optional here? there's a chance either links needs to be overwritten as diff path?

Reply:
can refer to the figma, because both /forms and /forms-selection have the same active tab state

Reply:
so basically if on the forms "page" then dont need specify, if on the form-selection "page" then specify. Like that cos SPA ma so i guess actually both refer to the same page